BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/fragment/PointsOfInterestMapFragment.java

```diff
@@ -0,0 +1,115 @@
+package com.izor066.android.blocspot.ui.fragment;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.MapView;
+import com.google.android.gms.maps.MapsInitializer;
+import com.google.android.gms.maps.model.BitmapDescriptorFactory;
+import com.google.android.gms.maps.model.CameraPosition;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.izor066.android.blocspot.BlocSpotApplication;
+import com.izor066.android.blocspot.R;
+import com.izor066.android.blocspot.api.model.PointOfInterest;
+
+import java.util.List;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+
+public class PointsOfInterestMapFragment extends Fragment {
+
+
+    MapView mapView;
+    private GoogleMap googleMap;
+
+    List<PointOfInterest> pointsOfInterest = BlocSpotApplication.getSharedDataSource().getAllPointsOfInterest();
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        Log.v("MapFragment", "OnCreateView");
+
+        View view = inflater.inflate(R.layout.fragment_points_of_interest_map, container, false);
+        mapView = (MapView) view.findViewById(R.id.mapView);
+        mapView.onCreate(savedInstanceState);
+        Log.v("MapFragment", "mapView.OnCreate");
+
+        mapView.onResume();
+        Log.v("MapFragment", "mapView.onResume");
+
+        try {
+            MapsInitializer.initialize(getActivity().getApplicationContext());
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
In Android `e.printStackTrace()` is an expensive method call. If you want to log what exception has been thrown it is advisable to use the Android log: 
`Log.e(TAG, "An exception occurred while initialising Maps", e)`.